opae.spec.in

```diff
@@ -217,3 +234,11 @@ ldconfig
 @CMAKE_INSTALL_PREFIX@/bin/hello_events
 
 %changelog
+* Mon Dec 17 2019 Korde Nakul <nakul.korde@intel.com> -1.4.0
```

Comment:
I think the changelog is generated by the rpm creation script.  No need to include this here.